readme.md

```diff
@@ -0,0 +1,191 @@
+# remark for Visual Studio Code
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+Visual Studio Code extension to format and lint markdown files with remark.
+
+## Contents
+
+*   [What is this?](#what-is-this)
+*   [When should I use this?](#when-should-i-use-this)
+*   [Use](#use)
+    *   [Formatting](#formatting)
+*   [Plugins](#plugins)
+*   [Syntax](#syntax)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## What is this?
+
+This project is a Visual Studio Code (VS Code) extension that you can use in
+your editor to inspect and change markdown files.
+This extension is built around remark, which is a very popular ecosystem of
+plugins that work with markdown.
+You can choose from the 150+ plugins that already exist or make your own.
+
+See [remark][] for info on what the remark ecosystem is.
+
+## When should I use this?
+
+You can use this extension if you want to check (lint) and format markdown
+files from within your editor.
+
+To configure this extension, you define your preferred markdown style in a
+configuration file (`.remarkrc`, `.remarkrc.js`, etc. or in `package.json`).
+This file is picked up by `vscode-remark` and other tools (useful for
+contributors that don’t use VS Code).
+
+The configuration file is also used by [`remark-cli`][remark-cli], which is
+recommended to be used alongside `vscode-remark`, as an npm script and/or in
+CI, to enforce the markdown style.
+
```

Comment:
Should be an install section here? Perhaps VSC comes with a CLI as well, to do that with a command, and/or perhaps an example of how to add it with the GUI?

Reply:
I thought I added one already, but I was wrong :thinking:. Anyway, there are multiple ways to get there. I added one method, as I don’t think it’s necessary to document all methods a VS Code extension can be installed.

Also the method I chose works for open source VS Code forks (That’s another issue I’ll create later).

Reply:
Linking to it might be useful as well? `vscode:extension/xxxx`? Or linking to the marketplace?:

````
[Get it from the VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=xxxx)

Or launch Quick Open (<kbd>Ctrl</kbd> + <kbd>P</kbd>) and run:

```sh
ext install unifiedjs.vscode-remark
```
````

Reply:
I didn’t know about that link format. It appears to work in the VS Code markdown preview. However, GitHub doesn’t render it.

I now modelled it after https://github.com/prettier/prettier-vscode#installation